Signal/src/view controllers/CallViewController.swift

```diff
@@ -246,8 +452,18 @@ class CallViewController: UIViewController {
         self.callStatusLabel.text = textForState
 
         // Show Incoming vs. (Outgoing || Accepted) call controls
-        callControls.isHidden = callState == .localRinging
-        incomingCallControls.isHidden = callState != .localRinging
+        let isRinging = callState == .localRinging
+        for subview in allControls() {
```

Comment:
Yeah, I was tempted to do that, but I didn't want to make the assumption that no controls showed up in both `incomingCallControls` and `ongoingCallControls`.  I was trying to future-proof this against later refactoring since I expect we'll continue to rework this view.